AndroidJavaClient/app/src/main/java/com/example/streamrjavaclienttest/MainActivity.java

```diff
@@ -0,0 +1,89 @@
+package com.example.streamrjavaclienttest;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.widget.TextView;
+
+import com.streamr.client.MessageHandler;
+import com.streamr.client.StreamrClient;
+import com.streamr.client.authentication.EthereumAuthenticationMethod;
+import com.streamr.client.options.StreamrClientOptions;
+import com.streamr.client.protocol.message_layer.StreamMessage;
+import com.streamr.client.rest.Stream;
+import com.streamr.client.subs.Subscription;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        String ethereumPrivateKey = "YOUR_ETHEREUM_PRIVATE_KEY";
```

Comment:
If the user runs the app before he puts valid values to `ethereumPrivateKey` and `streamId`, we could maybe show some friendly message in the UI? Now it shows two `NumberFormatException`s.